database/postgres/dal/news.py

```diff
@@ -27,3 +27,19 @@ def fetch_news(self, order_by=None, limit=None, filters=None):
     def get_news_by_id(self, blob_id: str):
         return self.session.query(News).filter(News.blob_id == blob_id).first()
 
+
+    def get_news_by_real_id(self, news_id: str):
+        return self.session.query(News).filter(News.news_id == news_id).first()
+
+
+    def get_sport_by_name(self, sport_name):
+        return self.session.query(Sport).filter_by(sport_name = sport_name).first()
+
+
+    def get_news_ids(self, time_limit):
+        return (
+            self.session.query(News.news_id)
+            .filter(News.save_at >= time_limit)
+            .all()
```

Comment:
What if we have thousands of news? Pagination is required.

However, that's no use to hardcode pagination in each method like this one inside every DAL we have.
We are getting very close to SearchEngine here, so let's discuss it on the meeting today

Reply:
Pagination?